### PR TITLE
feat(consultation-portal): SEO component

### PR DESCRIPTION
**Pull request by korsbacka**
# SEO og Header fyrir samráðsgatt

https://veflausnir.atlassian.net/browse/KAM-890

## What

Adding a seo title for all <Head> in pages

## Why

/

## Screenshots / Gifs

/

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/consultation-portal/components/SEO/SEO.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        property="og:url"
+        content={`https://samradsgatt.island.is/${url}`}
+        key="ogurl"
+      />
+      <meta property="og:image" content={image ?? image} key="ogimage" />
+      <meta property="og:site_name" content="Samráðsgátt" key="ogsitename" />
+      <meta
+        property="og:title"
+        content={`${title} | Samráðsgátt`}
+        key="ogtitle"
```
Comment (diljath) [inline apps/consultation-portal/components/SEO/SEO.tsx:23]: spurning hvort mætti snúa þessu við svo það komi "Samráðsgátt | title"
Það er þannig á gömlu síðunni

File: apps/consultation-portal/pages/um/index.tsx [hunk 2]
```diff
@@ -4,31 +4,35 @@ import React from 'react'
...
-          </GridColumn>
-          <GridColumn span={'3/12'}>
-            <RightSideColmn aboutHeadings={null}></RightSideColmn>
-          </GridColumn>
-        </GridRow>
-      </GridContainer>
-    </Layout>
+    <>
+      <SEO title="Um samráðsgátt" url="um" />
+      <Layout>
```
Comment (diljath) [inline apps/consultation-portal/pages/um/index.tsx:17]: Spurning hvort SEO componentin ætti ekki bara að vera undir Layout?
Og feeda þá inn title og url inn í Layout.

File: apps/consultation-portal/components/SEO/SEO.tsx (hunk 1 above)
Comment (korsbacka) [reply apps/consultation-portal/components/SEO/SEO.tsx:23]: jú meikar sense

File: apps/consultation-portal/pages/um/index.tsx (hunk 2 above)
Comment (korsbacka) [reply apps/consultation-portal/pages/um/index.tsx:17]: jú miklu betra þannig! :)

**Review by diljath — APPROVED** [review]

**Pull request merged**